linera-sdk-derive/src/utils.rs

```diff
@@ -14,7 +14,7 @@ pub fn concat(ident: &Ident, suffix: &str) -> Ident {
     transform_non_keyword_ident(ident, |s: String| format!("{}{}", s, suffix))
 }
 
-/// Applies a string transformation (`transform`) to the input `Ident`-
+/// Applies a string transformation (`transform`) to the input `Ident`.
 /// However, it will not apply the transform to rust keywords.
```

Comment:
Thanks! Maybe this should also be uppercase:

```suggestion
/// However, it will not apply the transform to Rust keywords.
```